pages/tutorial/dependencies.md

```diff
@@ -130,7 +130,7 @@ Our final main program looks like
 Again we run a quick check using fpm
 
 ```{code-block} text
-:emphasize-lines: 2, 14
+:emphasize-lines: 2, 13
```

Comment:
Here the stdlib usage needs to be changed as well

Reply:
Good catch